### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by paulstretenowich**

### CODE REVIEW

File: project_tracking/api/project.py [hunk 1]
```diff
@@ -463,13 +464,30 @@ def ingest_genpipes(project_id: str):
             return redirect(request.url)
 
         project_id_from_name = db_action.name_to_id("Project", ingest_data[vc.PROJECT_NAME].upper())
-        if project_id != project_id_from_name:
+        if [int(project_id)] != project_id_from_name:
```
Comment (poquirion) [inline project_tracking/api/project.py:467]: If project_id  is not castable,  it is goin to crash.

```
 >>> int('fasfqa')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'fasfqa'
```
We might whan to mitigate that.

File: project_tracking/db_action.py [hunk 2]
```diff
@@ -521,7 +521,7 @@ def create_project(project_name, fms_id=None, session=None):
     return session.scalars(select(Project).where(Project.name == project_name)).one()
 
 
-def ingest_run_processing(project_id, ingest_data, session=None):
+def ingest_run_processing(project_id: str, ingest_data, session=None):
```
Comment (poquirion) [inline project_tracking/db_action.py:524]: project_id should be a str or an int? This would have implication in many routes signature

Comment (paulstretenowich) [reply project_tracking/db_action.py:524]: How do we want IDs being int jsonified so string to be?

File: project_tracking/api/project.py (hunk 1 above)
Comment (paulstretenowich) [reply project_tracking/api/project.py:467]: But an id being such thing is supposed to break right?

**Review by poquirion — APPROVED** [review]
Tout semble ok

**Pull request merged**